improving.html

```diff
@@ -129,7 +129,7 @@ <h2 id="explore">What are the Issues?</h2>
 	<h2 id="scope">What is your Scope?</h2>
 	<p>You may not be able to address all the issues on every part of your website at once. To determine which parts you may need to improve right away, and which to address in later stages, consider the following:</p>
 	<ul>
-		<li><strong>Key tasks</strong>, such as registration, purchase, or checkout processes. Include all steps in each of these flows.</li>
+		<li><strong>Key tasks</strong>, such as registration, purchase, or checkout processes. Include all steps involved to complete key tasks.</li>
```

Comment:
Compromise: "Key tasks, such as registration, search, submit, or checkout processes. Include all steps involved to complete each task."

1. "submission" sounds a bit odd to me
2. We need to "include" not only "consider" (WCAG conformance)